gatsby-config.js

```diff
@@ -744,10 +748,6 @@ module.exports = {
                     {
                         title: "iOS App Extensions",
                         path: "documentation/user-guides/app-extension"
-                    },
-                    {
-                        title: "Tracking beacons",
-                        path: "documentation/user-guides/track-beacon"
```

Comment:
@calebk1 @swarna04 this change breaks the old link, I think we should be good for these small instances but let me know it you consider we should use redirects

Reply:
Following up on this one - decided we will create redirects for all these links once all the changes are merged into the resources-and-guides branch